packages/components/src/mobile/utils.native.js

```diff
@@ -0,0 +1,16 @@
+/**
```

Comment:
Hmmm, WDYT about renaming that file to something related to the function that is exported instead of utils? Maybe we could name it `src/mobile/layout-animation/index.native.js`?